### PR TITLE
added: Nirsoft/AdvancedRun/1.51

### PR DESCRIPTION
**Pull request by neomax7**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://api.github.com/repos/microsoft/winget-pkgs/pulls/95488)

### CODE REVIEW

**Comment by neomax7** [comment]
hello 
the manifest validates and it installs on my laptop without an error. 
is there anyway to check the reason why **winget-svc-validation (installer validation)** process failed here?
thanks in advance

File: manifests/n/NirSoft/AdvancedRun/1.51/NirSoft.AdvancedRun.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,17 @@
+# Created with YamlCreate.ps1 v2.2.1 $debug=NVS0.CRLF.7-3-1.Win32NT
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: NirSoft.AdvancedRun
+PackageVersion: 1.51
+InstallerLocale: en-US
+InstallerType: zip
+Installers:
+- Architecture: x64
```
Comment (Trenly) [inline manifests/n/NirSoft/AdvancedRun/1.51/NirSoft.AdvancedRun.installer.yaml:9]: Please also add the x86 version - https://www.nirsoft.net/utils/advancedrun.zip

**Review by Trenly — CHANGES_REQUESTED** [review]

File: manifests/n/NirSoft/AdvancedRun/1.51/NirSoft.AdvancedRun.installer.yaml (hunk 1 above)
Comment (neomax7) [reply manifests/n/NirSoft/AdvancedRun/1.51/NirSoft.AdvancedRun.installer.yaml]: added :)

**Review by stephengillie — APPROVED** [review]

**Pull request merged**